tests/benchmark_test_inputs/AB/mvs_config.json

```
@@ -0,0 +1,168 @@
+        }
+    },
+    "energyConversion": {
+        "solar_inverter_01": {
+            "age_installed": {
+                "unit": "year",
+                "value": 0
+            },
+            "capex_fix": {
+                "unit": "currency",
+                "value": 0.0
+            },
+            "capex_var": {
+                "unit": "currency/kW",
+                "value": 3000
+            },
+            "efficiency": {
+                "unit": "factor",
+                "value": 1.0
+            },
+            "energyVector": "Electricity",
+            "inflow_direction": "PV plant (mono)",
+            "installedCap": {
+                "unit": "kW",
+                "value": 0.0
+            },
+            "label": "Solar inverter (mono)",
+            "lifetime": {
+                "unit": "year",
+                "value": 15
+            },
+            "opex_fix": {
+                "unit": "currency/kW/year",
+                "value": 0.0
+            },
+            "opex_var": {
+                "unit": "currency/kWh",
+                "value": 0.0
+            },
+            "optimizeCap": {
+                "unit": "bool",
+                "value": true
+            },
+            "outflow_direction": "Electricity",
+            "type_oemof": "transformer",
+            "unit": "kW"
+        }
+    },
+    "energyProduction": {
+        "pv_plant_01": {
+            "age_installed": {
+                "unit": "year",
+                "value": 0
+            },
+            "capex_fix": {
+                "unit": "currency",
+                "value": 10000
+            },
+            "capex_var": {
+                "unit": "currency/unit",
+                "value": 7200
+            },
+            "energyVector": "Electricity",
+            "file_name": "production.csv",
+            "installedCap": {
+                "unit": "kWp",
+                "value": 0
+            },
+            "maximumCap": {
+                "unit": "kWp",
+                "value": 1000
+            },
+            "label": "PV plant (mono)",
+            "lifetime": {
+                "unit": "year",
+                "value": 30
+            },
+            "opex_fix": {
+                "unit": "currency/unit/year",
+                "value": 80
+            },
+            "opex_var": {
+                "unit": "currency/kWh",
+                "value": 0
+            },
+            "optimizeCap": {
+                "unit": "bool",
+                "value": true
+            },
+            "outflow_direction": "PV plant (mono)",
+            "type_oemof": "source",
+            "unit": "kWp"
+        }
+    },
+    "energyProviders": {},
+    "energyStorage": {},
+    "fixcost": {},
+    "project_data": {
+        "country": "NA",
+        "label": "project_data",
+        "latitude": "0",
+        "longitude": "0",
+        "project_id": "1",
+        "project_name": "Benchmark test for peak pricing case electricity bus",
```

```suggestion
        "project_name": "Benchmark test for replacing grid consumtion with PV generation",
```
This is what you would do here, rather.